griptape/memory/meta/action_subtask_meta_entry.py

```diff
@@ -15,6 +18,6 @@ class ActionSubtaskMetaEntry(BaseMetaEntry):
 
     type: str = field(default=BaseMetaEntry.__name__, kw_only=True, metadata={"serializable": False})
 
-    thought: str = field(kw_only=True, metadata={"serializable": True})
+    thought: Optional[str] = field(kw_only=True, metadata={"serializable": True})
```

Comment:
Don't know if we want this change. What was the pyright error this resolved?

Reply:
This change resolves  /Users/aodhanroche/Documents/GitHub/Griptaping/griptape/griptape/memory/task/task_memory.py:67:56 - error: Argument of type "str | None" cannot be assigned to parameter "thought" of type "str" in function "__init__"
    Type "str | None" cannot be assigned to type "str"
      "None" is incompatible with "str" (reportArgumentType)